src/components/TransferForm.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react'
+
+import { InputField } from '@components'
+import { TransferProps } from '@containers'
+import { formAlert } from '@helpers/alert'
+import { InputError, WalletLockError } from '@helpers/errors'
+import { Loader } from './Loader'
+import { PayeeSelector } from './PayeeSelector'
+
+export class TransferForm extends React.Component<TransferProps> {
+  constructor(props) {
+    super(props)
+  }
+
+  initTransfer = async (ev: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
Jeremy mentioned the other day that it might be good to show the name of the payee somewhere in this dialog, in case the user has forgotten who they are paying to. This might not be necessary now that there's the payee saving etc as it's probably more obvious who it's going to than it was when it was just a long public key string.... what do you think?

Reply:
Now that the the public key si always shown in the transfer form, I don't think this is an issue now.